<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettWindget</class>
 <widget class="QDialog" name="SettWindget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="Settings">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4" rowstretch="4,1,0,3" columnstretch="6,1,2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout" columnstretch="2,4,2,0">
       <item row="4" column="2">
        <widget class="QLabel" name="NewCurr">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Buttons">
         <property name="text">
          <string>Buttons:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="OpenCurr">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="OpenEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="CloseLable">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="CloseEdit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="NewEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="SaveEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="NewLable">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="SaveCurr">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="Current">
         <property name="text">
          <string>Current</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="SaveLable">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="CloseCurr">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="OpenLable">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="OpenApply">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="SaveApply">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="CloseApply">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="NewApply">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QRadioButton" name="AltChecker">
         <property name="text">
          <string>Alt</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="ModsLable">
         <property name="text">
          <string>Modifiers</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="ShiftChecker">
         <property name="text">
          <string>Shift</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="ApplyMod">
         <property name="text">
          <string>Apply modifier</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="AppChange">
         <property name="text">
          <string>Apply changes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="Reset">
       <property name="text">
        <string>Reset changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettWindget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettWindget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
